src/django_keycloak/services/oidc_profile.py

```diff
@@ -102,18 +114,29 @@ def update_or_create_user_and_oidc_profile(client, id_token_object):
     with transaction.atomic():
         UserModel = get_user_model()
 
+        user_query_field = getattr(settings, 'KEYCLOAK_USERNAME_FIELD', UserModel.USERNAME_FIELD)
         username_field = UserModel.USERNAME_FIELD
         email_field_name = UserModel.get_email_field_name()
 
+        user_identifier = id_token_object.get('preferred_username', id_token_object['sub'])
+        if hasattr(settings, 'KEYCLOAK_USERNAME_TOKEN_ATTRIBUTE'):
+            resolved = resolve_token_attribute(id_token_object,
+                                               settings.KEYCLOAK_USERNAME_TOKEN_ATTRIBUTE)
+            import sys
+            print(f'--> {settings.KEYCLOAK_USERNAME_TOKEN_ATTRIBUTE} => {resolved}', file=sys.stderr)
```

Comment:
Why not using logging?

Reply:
Whoopsie. The print shouldn't be in there.